packages/vitest-pool-workers/src/pool/index.ts

```diff
@@ -328,8 +329,37 @@ function fixupDurableObjectBindingsToSelf(
 	return result;
 }
 
+function getWranglerWorkerName(
+	relativeWranglerConfigPath?: string
+): string | undefined {
+	if (!relativeWranglerConfigPath) {
+		return undefined;
+	}
+	const wranglerConfigObject = experimental_readRawConfig({
+		config: relativeWranglerConfigPath,
+	});
+	return wranglerConfigObject.rawConfig.name;
+}
+
+function updateWorkflowsScriptNames(
+	runnerWorker: WorkerOptions,
+	wranglerWorkerName: string | undefined,
+	testWorkerName: string
+): void {
+	const workflows = runnerWorker.workflows;
+	if (!workflows || wranglerWorkerName === undefined) {
+		return;
+	}
+	for (const workflow of Object.values(workflows)) {
+		if (workflow.scriptName === wranglerWorkerName) {
+			workflow.scriptName = testWorkerName;
```

Comment:
Would it also be enough just to remove the `scriptName` since it would then default to the current worker?

Reply:
Do we have the same problem with normal service bindings?